audbcards/core/dataset.py

```diff
@@ -144,6 +144,35 @@ def languages(self) -> typing.List[str]:
         r"""Languages of the database."""
         return self.header.languages
 
+    @property
+    def iso_languages(self) -> typing.List[str]:
+        r"""Languages of the database."""
+        return self._map_iso_languages(self.languages)
+
+    @staticmethod
+    def _map_iso_languages(languages : typing.List[str]) -> typing.List[str]:
+        r"""Calculate ISO languages for a list of languages.
+
+        Args:
+            languages: list of languages as given in the header languages
+        Returns:
+            list of languages
+
+        Behavior: leaves languages intact if `audformat.utils.map_language`
+        raises ValueError
+
+        """
+        iso_languages = []
+        for lang in languages:
+            try:
+                iso_language = audformat.utils.map_language(lang)
+            except ValueError:
+                iso_language = lang
+
+            iso_languages.append(iso_language)
+
+        return list(set(iso_languages))
```

Comment:
As `set()` returns sequences in random order, I would propose to sort the output here:

```suggestion
        return sorted(list(set(iso_languages)))
```

Reply:
Makes sense. So you do not have reservations against the set representation? This precludes to inverse relationship back to the language property.  Suggestion is committed.